[2020-06-01 20:21:51,279] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_cvd.multitask2 2020-05-30T00:15:00+00:00 [queued]>
[2020-06-01 20:21:51,292] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_cvd.multitask2 2020-05-30T00:15:00+00:00 [queued]>
[2020-06-01 20:21:51,293] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 20:21:51,293] {taskinstance.py:867} INFO - Starting attempt 1 of 3
[2020-06-01 20:21:51,293] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 20:21:51,311] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): multitask2> on 2020-05-30T00:15:00+00:00
[2020-06-01 20:21:51,314] {standard_task_runner.py:53} INFO - Started process 15130 to run task
[2020-06-01 20:21:51,388] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: test_dag_cvd.multitask2 2020-05-30T00:15:00+00:00 [running]> nineleaps
[2020-06-01 20:21:51,401] {logging_mixin.py:112} INFO - Hello-World
[2020-06-01 20:21:51,401] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-06-01 20:21:51,405] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=test_dag_cvd, task_id=multitask2, execution_date=20200530T001500, start_date=20200601T145151, end_date=20200601T145151
[2020-06-01 20:22:01,257] {logging_mixin.py:112} INFO - [2020-06-01 20:22:01,256] {local_task_job.py:103} INFO - Task exited with return code 0
[2020-06-01 20:56:41,915] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_cvd.multitask2 2020-05-30T00:15:00+00:00 [queued]>
[2020-06-01 20:56:41,925] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: test_dag_cvd.multitask2 2020-05-30T00:15:00+00:00 [queued]>
[2020-06-01 20:56:41,925] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 20:56:41,925] {taskinstance.py:867} INFO - Starting attempt 1 of 3
[2020-06-01 20:56:41,925] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2020-06-01 20:56:41,942] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): multitask2> on 2020-05-30T00:15:00+00:00
[2020-06-01 20:56:41,946] {standard_task_runner.py:53} INFO - Started process 19495 to run task
[2020-06-01 20:56:42,030] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: test_dag_cvd.multitask2 2020-05-30T00:15:00+00:00 [running]> nineleaps
[2020-06-01 20:56:42,040] {logging_mixin.py:112} INFO - Hello-World
[2020-06-01 20:56:42,040] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-06-01 20:56:42,043] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=test_dag_cvd, task_id=multitask2, execution_date=20200530T001500, start_date=20200601T152641, end_date=20200601T152642
[2020-06-01 20:56:51,904] {logging_mixin.py:112} INFO - [2020-06-01 20:56:51,904] {local_task_job.py:103} INFO - Task exited with return code 0
